app/src/main/java/de/westnordost/streetcomplete/statistics/UploadsCounter.java

```diff
@@ -0,0 +1,53 @@
+package de.westnordost.streetcomplete.statistics;
+
+import android.content.Context;
+import android.os.AsyncTask;
+import android.util.AttributeSet;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.Injector;
+import de.westnordost.streetcomplete.data.QuestStatus;
+import de.westnordost.streetcomplete.data.osm.persist.OsmQuestDao;
+
+public class UploadsCounter extends android.support.v7.widget.AppCompatTextView {
```

Comment:
The name for the new class is confusing. After all, this is the counter of answers that have _not_ been uploaded yet.

The other (original) counter counts the number of quest answers that have been uploaded already. This one counts the number of newly added quest answers that are pending to be uploaded. The class and field names should reflect that.

Reply:
Yeah, I did feel there was something off. How about if I renamed both the classes. UnsyncedAnswersCounter and UploadedAnswersCounter? Both their field names can be answeredQuests since the class name will specify what kind of answered quests they contain.

Reply:
These names would work, yes.
> answeredQuests

I meant the names of the fields in MainActivity.